docs/test-pr-guide.md

```diff
@@ -23,6 +23,40 @@ git pull https://github.com/<contributor>/mentorship-backend/ <branch-name>
 
 8. Get screenshots/gifs of before and after the fix and attach them to the PR comment.
 
+9. If the testing is unsuccessful and fixes are needed change then remove the label _Needs Testing_ and add _Needs Review_.
+
+10. If the testing is successful and no improvements needed then remove the label _Needs Testing_ and add _Ready to Merge_.
+
+<h2>Template to report PR testing results</h2>
+This template can be use to add a review comment to a PR after testing is done. It can used used irrespective of success or failure of testing.
+
+The changes made in this PR was tested locally. Folowing are the results:
```

Comment:
I would wrap the template with "```" in a code block, such as:

```
The changes made in this PR was tested locally. Folowing are the results:

(...)

5. Status of PR Changed to: _Needs Review_/_Ready to Merge_.
```